journalpump/journalpump.py

```diff
@@ -495,6 +496,14 @@ def value_func(tags, f=f, args=args):  # pylint: disable=undefined-variable
 
             yield output
 
+    def _build_secret_filters(self, config):
+        secret_filters = config.get("secret_filters")
```

Comment:
Could use `config.get("secret_filters", [])`. Can then check the type of `secret_filters` to ensure it is a list before iteration.